### DISCUSSION TITLE
webhook to awx prefix

### DISCUSSION THREAD

**Discussion by Dees7** [netbox-community/netbox-docker#528] [Getting Help] [answered]
hello.
I have awx with webhook to run deploy.  And I'd like to pass netbox request body to awx. 
AWX requere prefix "extra_vars" in POST like
```{"extra_vars":{"event": "updated", "username": "admin", ... , "tags": []}}```

In webhook in Netbox If "body template" is empty it send all.  But if no "extra_vars" prefix AWX skip vars
```{"event": "updated", "username": "admin", ... , "tags": []}```

I can write template like {"extra_vars":{{data}} .....  prechange  postchange username } but I'd like only to add "extra_vars" before  whole  and every request body.

Is there any variable for a whole body? or Can I add prefix in any ways?

**Comment by cimnine** [comment, marked as answer]
I think this is a question [for the NetBox project](https://github.com/netbox-community/netbox); this project here is just about the Docker distribution of NetBox.
